File: Programming Fundamentals with Python/Functions/add_and_subtract.py

def sum_numbers(num1, num2):
    return num1 + num2

def subtract(sum, num3):
    return sum - num3


num1 = int(input())
num2 = int(input())
num3 = int(input())

sum = sum_numbers(num1, num2)
result = subtract(sum, num3)
print(result)

File: Programming Fundamentals with Java/Methods/AddAndSubtract.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class AddAndSubtract 
{
    public static Double addAndSubtract(double num1, double num2, double num3)
    {
        return (num1 + num2) - num3;
    }

    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);

        double num1 = Double.parseDouble(input.nextLine());
        double num2 = Double.parseDouble(input.nextLine());
        double num3 = Double.parseDouble(input.nextLine());

        double result = addAndSubtract(num1, num2, num3);
        System.out.println(new DecimalFormat("0.####").format(result));
    }
}
